content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -9,71 +9,140 @@ weight: 22033
 
-After configuration, restart your control plane to start the server.
+### Control Plane repository
+
+#### AWS S3
+
+{{< alert warning >}}
+Control plane with private repository needs AWS permissions `s3:PutObject`, `s3:DeleteObject` and `s3:GetObject` on the bucket.
+{{< /alert >}}
+
+Once it is done, add the private repository configuration section in your [control plane configuration]({{< ref "../introduction" >}}) file:
+
+```bash
+control-plane {
+  repository {
+    # S3 Bucket configuration
+    type = "aws"
+    bucket = "bucket-name"
+    path = "folder/to/upload" # (optional, default: root)
+  }
+}
+```
+
+This configuration includes the following parameters:
+- **bucket**: The name of the bucket where packages are uploaded to on AWS S3.
+- **path:** The path of a folder in AWS S3 bucket. (optional)
+
+#### GCP Cloud Storage
+
+{{< alert warning >}}
+Control plane with private repository needs GCP service account role with permissions `storage.objects.create`, 
+`storage.objects.delete` and `iam.serviceAccounts.signBlob` on the bucket.
+{{< /alert >}}
+
+```bash
+control-plane {
+  repository {
+    # Cloud Storage Bucket configuration
+    type = "gcp"
+    bucket = "bucket-name"
+    path = "folder/to/upload" # (optional, default: root)
+  }
+}
+```
+
+This configuration includes the following parameters:
+- **bucket**: The name of the bucket where packages are uploaded to on GCP Cloud Storage.
+- **path:** The path of a folder in Cloud Storage bucket. (optional)
 
 ## Usage
 
-You have to use Gatling plugins to create and upload your private packages. See Gatling documentation:
-* [Maven plugin](https://gatling.io/docs/gatling/reference/current/extensions/maven_plugin/)
-* [Gradle plugin](https://gatling.io/docs/gatling/reference/current/extensions/gradle_plugin/)
-* [SBT plugin](https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/)
+After configuration, restart your control plane to start the server.
+
+### Creating a Private Package
+
+To create a private package, you can use Gatling Enterprise Cloud's public API. Follow these steps:
+
+1. Obtain an API Token with `Configure` permission for the relevant team.
+2. Navigate to the Swagger package section for further instructions on creating a private package.
+3. When creating the package, ensure you select a private storage type.
+4. Use Gatling plugins to create and upload your private packages. You can refer to the Gatling documentation for specific plugin usage:
+  - [Maven plugin](https://gatling.io/docs/gatling/reference/current/extensions/maven_plugin/)
+  - [Gradle plugin](https://gatling.io/docs/gatling/reference/current/extensions/gradle_plugin/)
+  - [SBT plugin](https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/)
+5. For each plugin, you will need to configure it with the following details:
+  - Your API token with "configure" permission for the private package team.
```

Comment:
earlier `Configure` is used, and here is it "configure." I like the first and it is most consistent with how it appears in the product.